scripts/release-on-comment.py

```diff
@@ -168,6 +171,22 @@ def trigger_release(payload_path: Path, token: str) -> None:
         )
         print_and_exit(f"{Fore.RED}{e}")
 
+    if error_contents:
```

Comment:
I suggest one of two things:

1. Inline this inside of the `except CallProcessError` branch, instead of putting in the end and using `error_contents` variable.

2. Or, if you want to avoid duplication, make the `except Exception as e` branch use the `error_contents` variable as well (would be `error_contents = str(e)`).

Reply:
Oh I meant to suggest 2), I probably wasn't clear enough.